tests/unit/code/test_files.py

```diff
@@ -14,6 +17,15 @@ def test_files_fix_ignores_unsupported_extensions():
     assert not files.apply(path)
 
 
+def test_files_fix_ignores_unsupported_language():
+    languages = create_autospec(Languages)
+    files = Files(languages)
+    # pylint: disable=protected-access
```

Comment:
```suggestion
```
this fails `no-lint-disabled` check:
<img width="835" alt="image" src="https://github.com/databrickslabs/ucx/assets/259697/2eee1f47-2449-4ed8-84d0-0831de54abd0">
<img width="549" alt="image" src="https://github.com/databrickslabs/ucx/assets/259697/c01db6b3-cc3b-438a-ba09-5466846cfaa5">

left a suggestion on how to achieve the same intent, but without compromising access "guards". 

P.S. `acceptance` would run once you're onboarded and part of `databrickslabs` gh org

Reply:
That code was there.
I removed it because local `make fmt` said it isn't necessary
But then CI complained about private access... weird.
Since it's in no way related to this PR, I suggest keeping the original code (i.e. keeping line 23)

Reply:
If I don't put this, then `make fmt` fails. Is that preferable ?